docs/philosophy.md

```diff
@@ -98,3 +98,11 @@ which must be used when recording physical information into [Attributes](../spec
 
 However, uncertainty quantification can be challenging and is often impossible in hindsight.
 Therefore, `nominal` values are permitted to express "It was nominally X, but we don't know how certain we are about it."
+
+
+#### Material history is chronological
+
+Starting from a given material, one can trace back the process that created the material, 
+the ingredients that went into the process, the materials that those ingredients were derived from, and so on.
+This history is chronological: for example, the ingredients that are involved in a process must exist *before* the process occurs.
+As a consequence, there can be no cycles in a material history.
```

Comment:
I can't figure out how to do multiline-suggestions, and spawning a PR seems a bit much.  I'd propose this reads:

For any physical object, one can trace back the series of processes that generated it and the ingredient materials that went into those processes.  
Formally, such a chronology represents a directed acyclic graph, where causal structure leads naturally to a topological ordering for nodes.  
Following this formalism and the common-sense idea that time moves in one direction only, a material history cannot contain a cycle, i.e., following references in the order material->process->ingredient->material will never see the same node twice.